journalbeat/tests/system/config/journalbeat.yml.j2

```diff
@@ -2,7 +2,10 @@
 journalbeat.inputs:
 - paths: [{{ journal_path }}]
   seek: {{ seek_method }}
-  matches: [{{ matches }}]
+  {% if seek_method == 'cursor' %}
```

Comment:
For limiting surprises instead of checking for `seek_method == "cursor"` it would be more flexible and explicit to use `if cursor_seek_fallback`

Reply:
I wrote `if cursor_seek_fallback and seek_method == 'cursor'` to not to add that line when some other seek method is configured.

Reply:
What I meant by that is when you write your tests you want a direct relationship to what you define in the dictionary in python and what will be generated in the YAML config. When you use `if cursor_seek_fallback and seek_method == 'cursor'` you make the assumption that I might have made a mistake by configuring `seek_method` to another value than "cursor". 

The problem by doing that is I won't know it without either looking at the YAML or in the output log.